### PR TITLE
WT-5446 Fix not writing to lookaside

### PR DESCRIPTION
**Pull request by quchenhao**

### CODE REVIEW

**Comment by quchenhao** [comment]
retest this please

**Comment by quchenhao** [comment]
I think there is actually no bug.

File: test/csuite/wt4105_large_doc_small_upd/main.c [hunk 1]
```diff
@@ -132,7 +132,8 @@ main(int argc, char *argv[])
             modify_entry.data.size = strlen(modify_entry.data.data);
             modify_entry.offset = offset;
             modify_entry.size = modify_entry.data.size;
-            (void)alarm(1);
+            /* FIXME-PM-1521: extend timeout to pass the test */
+            (void)alarm(5);
```
Comment (tetsuo-cpp) [inline test/csuite/wt4105_large_doc_small_upd/main.c]: Do we understand why this is necessary now?

Comment (quchenhao) [reply test/csuite/wt4105_large_doc_small_upd/main.c]: I think we incur more overhead in lookaside eviction.

**Comment by quchenhao** [comment]
@tetsuo-cpp I understand why they are failing and already created tickets. See WT-5451, WT-5448, and WT-5452

**Pull request merged**